File: CPP_Programming/Day_5/program32.cpp
/*  
    Description : Write a program that accepts frequency & prints
                  the below pattern : 
    Input : 4
    Output :-4 -3 -2 -1 0 1 2 3 4
*/

#include<iostream>
using namespace std;

class Pattern
{
    public :
        int iNo;

    Pattern(int iNo)
    {
        this->iNo = iNo;
    }

    void Display()
    {
        int iCnt = 0;
        
        for(iCnt = -iNo ; iCnt <= iNo ; iCnt++)
        {
            cout<<iCnt<<"\t";
        }

    }

};

int main()
{
    int iValue = 0;

    cout<<"Enter the frequency : ";
    cin>>iValue;

    Pattern pobj(iValue);

    pobj.Display();


    return 0;
}

File: Java_Programming/Day_5/program32.java
/*  
    Description : Write a program that accepts frequency & prints
                  the below pattern : 
    Input : 4
    Output :-4 -3 -2 -1 0 1 2 3 4
*/

import java.util.*;

class Pattern
{
    public int iNo;

    Pattern(int iNo)
    {
        this.iNo = iNo;
    }

    public void Display()
    {
        int iCnt = 0;

        for(iCnt = -iNo ; iCnt <= iNo ; iCnt++)
        {
            System.out.print(iCnt + "\t");
        }
    }

}

class program32
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter the frequency : ");
        int iValue = sobj.nextInt();

        Pattern pobj = new Pattern(iValue);

        pobj.Display();

        sobj.close();
    }
}